samples/net/sockets/sntp_client/src/main.c

```diff
@@ -33,17 +33,20 @@ void main(void)
 	rv = sntp_init(&ctx, (struct sockaddr *) &addr,
 		       sizeof(struct sockaddr_in));
 	if (rv < 0) {
-		LOG_ERR("Failed to init sntp ctx: %d", rv);
+		LOG_ERR("Failed to init SNTP IPv4 ctx: %d", rv);
 		goto end;
 	}
 
-	rv = sntp_request(&ctx, K_FOREVER, &epoch_time);
+	LOG_INF("Sending SNTP IPv4 request...");
+	rv = sntp_request(&ctx, 4000, &epoch_time);
```

Comment:
For timeout, better use `K_SECONDS(4)` instead, preferably via a define.

Reply:
Used `K_SECONDS(4)`. As these are supposed to be learning samples, using too much defines would only make them more complicated to follow (and yes, there're different delays, e.g. IPv4 case vs IPv6 case one; better just to have them inline).

Reply:
> As these are supposed to be learning samples, using too much defines would only make them more complicated to follow

I am disagreeing here. For the novice user reading the code, it is much easier if the parameter says `SNTP_TIMEOUT` than `K_SECONDS(4)`. Not a big issue thou, it is still better now than before.